ciao-storage/ceph.go

```diff
@@ -166,16 +170,22 @@ func (d CephDriver) MapVolumeToNode(volumeUUID string) (string, error) {
 // UnmapVolumeFromNode unmaps a ceph volume from a local device on a node.
 func (d CephDriver) UnmapVolumeFromNode(volumeUUID string) error {
 	args := append(d.getCredentials(), "unmap", volumeUUID)
-	return exec.Command("rbd", args...).Run()
+	cmd := exec.Command("rbd", args...)
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("Error when running: %v: %v: %s", cmd.Args, err, out)
+	}
+	return nil
 }
 
 // GetVolumeMapping returns a map of volumeUUID to mapped devices.
 func (d CephDriver) GetVolumeMapping() (map[string][]string, error) {
 	args := append(d.getCredentials(), "showmapped", "--format", "json")
 	cmd := exec.Command("rbd", args...)
-	data, err := cmd.Output()
+	data, err := cmd.CombinedOutput()
```

Comment:
This is changing the behaviour of the function.  The old version was reading the first line of Stdout.  The new code now reads the first line of combined Stdout and Stderr which might not be the same thing.  Probably best to continue to use Output, but to retrieve the error information by

err.(*ExitError).Stderr